src/test/dialogs.test.js

```diff
@@ -0,0 +1,209 @@
+var expect = require("chai").expect;
+var React = require("react/addons");
+var TestUtils = React.addons.TestUtils;
+
+var DialogActions = require("../js/actions/DialogActions");
+var DialogEvents = require("../js/events/DialogEvents");
+var DialogStore = require("../js/stores/DialogStore");
+
+var expectAsync = require("./helpers/expectAsync");
+
+describe("Dialog store", function () {
+
+  describe("alert", function () {
+
+    it("show sends correct message", function (done) {
+      DialogActions.alert("test alert");
+
+      DialogStore.once(DialogEvents.ALERT_SHOW, function (message) {
+        expectAsync(function () {
+          expect(message).to.equal("test alert");
+        }, done);
+      });
+    });
+
+    it("show sends correct dialogId", function (done) {
```

Comment:
we could have tested `message` and `dialogId` together as well, but this is also fine!